src/SmartTransactionsController.ts

```diff
@@ -520,6 +608,22 @@ export default class SmartTransactionsController extends BaseController<
         },
       },
     });
+
+    if (networkClientId) {
```

Comment:
does it make a difference if we populated feesByChainId even if networkClientId isn't provided?

Reply:
Yeah I suppose not. Presumably we won't start accessing that state until networkClientId is the passed everywhere and the globally selected network version of the state is deprecated. But yeah we can add it to the `feesByChainId` regardless for now.